src/systemctl/systemctl.h

```diff
@@ -104,6 +104,8 @@ extern bool arg_mkdir;
 extern bool arg_marked;
 extern const char *arg_drop_in;
 extern ImagePolicy *arg_image_policy;
+extern bool arg_list_by_slice;
+extern Hashmap *slice_unit_map;
```

Comment:
nope, we generally do not want global variables in our codebase, with some very specific exceptions, i.e. if the variables are directly initialized from data parsed from inherently process global concepts, such as env vars and command line options.

For the rest, please pass the variables through and define them with a clear context